release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -438,6 +438,24 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-networking_{context}"]
 === Networking
 
+[id="ocp-ptp-fast-events-rest-api-v2-available_{context}"]
+==== New PTP fast events REST API v2 available
+
+A new PTP fast events O-RAN v3 compliant REST API v2 is available.
+Now, you can develop PTP event consumer applications that receive host hardware PTP events directly from the PTP Operator managed pod via REST API.
+The PTP fast events REST API v1 will be deprecated in a future release.
+
+[NOTE]
+====
+In the link:https://orandownloadsweb.azurewebsites.net/download?id=344[_O-RAN O-Cloud Notification API Specification for Event Consumers 3.0_] specification, the resource is defined as a hierarchical path for the subsystem that produces the notifications.
+This implies that a subscription to a higher hierarchy path includes subscriptions for all lower hierarchy resources contained in the path.
```

Comment:
Could rewrite to avoid ambiguous pronoun.

Reply:
Good catch! Wonder could we make a Vale rule for this. Would be tricky. https://github.com/redhat-documentation/vale-at-red-hat/issues/881